test/mocks/deps/one.js

```diff
@@ -14,3 +14,16 @@ let a = require('@a/package')
 let b = require('@b/package/method')
 let c = require('c/specific/method')
 let d = require('d')
+
+// Minifiers may redefine variables to save space
+let missingInit
+missingInit = 'ignoredmodule'
+require(missingInit)
+
+// When minifiers redefine variables, the original might be an invalid module string
+let requiredThing = 'some thing '
+requiredThing = 'validmodule'
+require(requiredThing)
+
+// There may be a function called require that's not the real thing
+require(1234)
```

Comment:
Is this a case that we really ought to support? If someone is writing Node.js code that breaks CJS, I don't see any responsibility on our part to aid that.

Reply:
I agree. Though it's entirely possible that someone created a function called require, but I won't die on that hill :)